src/components/PatchesPage/ListArea.tsx

```diff
@@ -17,7 +17,8 @@ export const ListArea: React.FC<{
   >;
   patches?: PatchesPagePatchesFragment;
   error?: ApolloError;
-}> = ({ patches, error, analyticsObject }) => {
+  type: string;
```

Comment:
can we use a typescript type instead of string? I think we can replace `string` with  `"patch" | "user` or replace it with `PageType` where `type PageType = "patch" | "user"`